src/callRoutines.c

```diff
@@ -946,7 +946,7 @@ void runThresholdInputLatLons(char *MODEL_VERSION, char *OUTPUT_DIR, gen_extract
     }
     else if (strcmp(GEN_EXTRACT_VELO_MOD_CALL.VS_TYPE, "Z2.5") == 0)
     {
-        MODEL_EXTENT->Zmax = 12; // 10km downwards
+        MODEL_EXTENT->Zmax = 20; // 10km downwards
```

Comment:
10km at 0.5km increments?

Reply:
20km deep at 0.05km increments, so 400 points total now - increased from 240.

Reply:
In summary, update your comments :P

Reply:
TBH, I don't really understand the comments. And going back in time they don't make much sense either... 
![image](https://user-images.githubusercontent.com/25143301/129142554-99ddc4f1-873c-43ff-81a6-2ceb33851c7e.png)

Reply:
![image](https://user-images.githubusercontent.com/46693562/129143210-51d79ac4-0ef1-4670-b5bc-1e1ad39191ca.png)
There should be some method to the madness...

Reply:
@thomsonem What in tarnation?